plugins/scaffolder/src/plugin.tsx

```diff
@@ -96,6 +96,7 @@ export const scaffolderPlugin = createPlugin({
     registerComponent: registerComponentRouteRef,
     viewTechDoc: viewTechDocRouteRef,
   },
+  featureFlags: [{ name: 'experimental-scaffolder-templates' }],
```

Comment:
Think you can drop this, and we optionally register it if it's enabled.

Reply:
I need a tip how to do it. 
We need an access to `featureFlagApi` to register it, so it can be done only inside the component, but all components are lazy. And we are loading them depends which route to open.
So that means that we can't add this logic to scaffolder plugin, because if user goes directly to `http://localhost:3000/settings/feature-flags` we will not register the flag yet.